### DISCUSSION TITLE
What are the Keys bound to in the default firmware?

### DISCUSSION THREAD

**Discussion by JackWitherell** [cardonabits/haxo-hw#48] [Q&A] [answered]
I'm having a hard time figuring out how to use more than the 6 keys in the center row (234567 in this pic).
I was able to figure out how to switch instruments from the instructions in the readme, but holding down most other keys and blowing into the haxaphone just mutes all audio.
![fingering-matrix](https://github.com/cardonabits/haxo-hw/assets/4626546/c70ec9ec-9d97-42ff-9a75-2689b305cb08)

What are the keybinds of the haxophone in the default configuration?

The first one i'd like to know about is the top key. In the video in the readme linked [here,](https://github.com/cardonabits/haxo-hw#what-is-a-haxophone) at around 52 seconds in, a high note plays. I can clearly see the top key being pressed. However, whenever I hold that key and blow, no sound comes out at all. Practically no combination of keys makes sense other than the main 6 I mentioned.
How do I hit that high key as shown in the video?

In general, however, I'm really wondering what all of the other keys than those main 6 do, since whenever I'm holding them, the haxaphone stays muted. Sometimes, it even 'latches' into muted mode until I wait a few seconds or press a bunch of other keys at random to unlock it.

**Comment by jcard0na** [comment, marked as answer]
Hi Jack,

See [this discussion](https://github.com/cardonabits/haxo-rs/discussions/18) for a couple of fingering visualizers for the haxophone and let me know if that helps.

This is the default mapping:  https://github.com/cardonabits/haxo-rs/blob/main/notemap.json file, which you would need to modify for a different configuration (not very understandable, but the mapping viewers linked on that discussion thread can help decipher it).

Cheers,
